crypto/fipsmodule/pbkdf/pbkdf.c

```diff
@@ -72,6 +73,18 @@ int PKCS5_PBKDF2_HMAC(const char *password, size_t password_len,
   HMAC_CTX hctx;
   HMAC_CTX_init(&hctx);
 
+  // We have to avoid the underlying SHA services updating the indicator
+  // state, so we lock the state here.
+  FIPS_service_indicator_lock_state();
+
+  // FIPS 140 parameter requirements, per NIST SP800-132:
```

Comment:
Do we need to keep this comment here or it will go in the service indicator check?

Reply:
Oh, good point. I wanted to keep it in the code somewhere, I'll see if there's a spot in the service indicator that works.